yaml_files/simphony_metadata.yml

```diff
@@ -445,38 +452,71 @@ CUDS_KEYS:
   # Conditions on Boundaries
   ######################################################
   DIRICHLET:
-    definition: Dirichlet boundary condition, specify the value the solutions takes on the boundary of the domain
+    definition: Dirichlet boundary condition to specify the value the solutions takes on the boundary of the domain.
     parent: CUBA.CONDITION
-    models: [CUBA.CONTINUUM]
-    CUBA.VARIABLE:
-      shape: (:)
-      default: []
     CUBA.MATERIAL:
-      shape: (:)
-      default: []
+      shape: [1]
+
+  CONSTANT_VELOCITY_CONDITION:
+    definition: Constant velocity condition
+    parent: CUBA.DIRICHLET
+    models: [CUBA.CONTINUUM]
+    CUBA.VELOCITY:
+      shape: [1]
+
+  CONSTANT_PRESSURE_CONDITION:
+    definition: Constant pressure condition
+    parent: CUBA.DIRICHLET
+    models: [CUBA.CONTINUUM]
+    CUBA.PRESSURE:
+      shape: [1]
+
+  CONSTANT_VOLUME_FRACTION_CONDITION:
+    definition: Constant volume fraction condition
+    parent: CUBA.DIRICHLET
+    models: [CUBA.CONTINUUM]
+    CUBA.VOLUME_FRACTION:
+      shape: [1]
 
   NEUMANN:
-    definition: Neumann boundary condition, it specifies the values that the derivative of a solution is to take on the boundary of the domain.
+    definition: Neumann boundary condition, it specifies the values that the derivative of a solution with respect to the variable is to take on the boundary of the domain.
     parent: CUBA.CONDITION
-    models: [CUBA.CONTINUUM]
-    CUBA.VARIABLE:
-      shape: (:)
-      default: []
     CUBA.MATERIAL:
-      shape: (:)
-      default: []
+      shape: [1]
 
-  EMPTY:
-    parent: CUBA.CONDITION
-    definition: Empty boundary condition
+  ZERO_GRADIENT_VELOCITY_CONDITION:
+    definition: Zero gradient velocity condition
+    parent: CUBA.NEUMANN
     models: [CUBA.CONTINUUM]
-    CUBA.VARIABLE:
-      shape: (:)
+    CUBA.VELOCITY:
+      shape: [1]
+
+  ZERO_GRADIENT_PRESSURE_CONDITION:
+    definition: Zero gradient pressure condition
+    parent: CUBA.NEUMANN
+    models: [CUBA.CONTINUUM]
+    CUBA.PRESSURE:
+      shape: [1]
+
+  ZERO_GRADIENT_VOLUME_FRACTION_CONDITION:
+    definition: Zero gradient volume fraction condition
+    parent: CUBA.NEUMANN
+    models: [CUBA.CONTINUUM]
+    CUBA.VOLUME_FRACTION:
```

Comment:
(I correct my previous reply!) good point but the material is already a property of the parent CUBA.NEUMANN and therefore the VOLUME_FRACTION is sufficient.